parm/config/gfs/config.com

```diff
@@ -96,5 +96,6 @@ declare -rx COM_ICE_GRIB_GRID_TMPL=${COM_ICE_GRIB_TMPL}'/${GRID}'
 
 declare -rx COM_CHEM_HISTORY_TMPL=${COM_BASE}'/model_data/chem/history'
 declare -rx COM_CHEM_ANALYSIS_TMPL=${COM_BASE}'/analysis/chem'
+declare -rx COM_CHEM_BMAT_TMPL=${COM_BASE}'/bmatrix/chem'
```

Comment:
I think the root should probably still be `${COM_BASE}'/analysis/chem'`, since the b-matrix is part of the analysis and we want to limit the number of "top" level directories. But it can still be a subdirectory within that.

Reply:
Is it okay though that there will be a mismatch of cycles with that approach? For example, the aerosol analysis job would be COMOUT_CHEM_ANALYSIS=gdas.20240717/18/analysis/chem but would need COMIN_CHEM_BMAT=gdas.20240717/12/analysis/chem/bmatrix

Reply:
The DA system has to read data from the previous cycle all the time, so I don't see a problem. That's why cycle info doesn't get substituted in until the template is filled: sometime we need the current cycle, sometimes we need the previous cycle (or even further back for some jobs).

Reply:
Ok, I'll make that change.